datasync/chaindatafetcher/kas/repository_token_transfer.go

```diff
@@ -0,0 +1,129 @@
+package kas
+
+import (
+	"fmt"
+	"github.com/klaytn/klaytn/blockchain"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/common"
+	"math/big"
+	"strings"
+)
+
+var tokenTransferEventHash = common.HexToHash("0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef")
+
+// splitToWords divides log data to the words.
+func splitToWords(data []byte) []common.Hash {
+	var words []common.Hash
+	for i := 0; i < len(data); i += common.HashLength {
+		words = append(words, common.BytesToHash(data[i:i+common.HashLength]))
+	}
+	return words
+}
+
+// wordToAddress trims input word to get address field only.
+func wordToAddress(word common.Hash) common.Address {
+	return common.BytesToAddress(word[common.HashLength-common.AddressLength:])
+}
+
+// transformsLogsToTokenTransfers converts the given event into Klaytn Compatible Token transfers.
+func transformsLogsToTokenTransfers(event blockchain.ChainEvent) []*KCTTransfer {
```

Comment:
How about changing `transformsLogsToTokenTransfers` to `transformLogsToTokenTransfers` or `transformEventToTokenTransfers`

Reply:
This look confusing.. 😅 
It is hard to distinguish between `transformsLogsToTokenTransfers` and `transformLogToTokenTransfer`.

Reply:
Removed `s`! 5a41a8aabea1abca07527c77b39da4be05d8c2c8